tests/models/semantic_role_labeller_test.py

```diff
@@ -0,0 +1,57 @@
+import numpy
+
+from allennlp.common import Params
+from allennlp.common.tensor import arrays_to_variables
+from allennlp.data import Vocabulary
+from allennlp.data.dataset_readers import SrlReader
+from allennlp.data.fields import TextField, IndexField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.models.semantic_role_labeller import SemanticRoleLabeller
+from allennlp.testing.test_case import AllenNlpTestCase
+
+
+class SemanticRoleLabellerTest(AllenNlpTestCase):
```

Comment:
Huh. That must have fallen through the cracks of my simple tagger and related PRs. I'll add it.

Regarding your second point, I think so. adding a backward pass to the `test_model_loads_and_saves` function would be easy though, just a case of adding `loss.backward()`. I'll also add this.